Sources/PrivoSDK/api/Rest.swift

```diff
@@ -48,6 +52,8 @@ protocol Restable {
     func processRecheck(data: RecheckStatusRecord) async throws -> AgeGateActionResponse
     func trackCustomError(_ errorDescr: String)
     func sendAnalyticEvent(_ event: AnalyticEvent)
+    func registerParentAndChild(_ parentChildPair: ParentChildPair, _ token: String) async throws -> RegisterResponse
+    func getP3Token(_ clientId: String, _ clientSecret: String) async throws -> TokenResponse
```

Comment:
1) why getP3Token is part of URL extension? Do not all urs will be able to call getP3Token
2) getP3Token looks like a strange naming, let's try to rename it

Reply:
* Discussed and decided to change request to gw token *